src/store_client.cc

```diff
@@ -527,12 +554,18 @@ store_client::readBody(const char *, ssize_t len)
              * copyInto.offset includes headers, which is what mem cache needs
              */
             int64_t mem_offset = entry->mem_obj->endOffset();
-            if ((copyInto.offset == mem_offset) || (parsed_header && mem_offset == rep->hdr_sz)) {
+            if ((copyInto.offset == mem_offset) || (headerParsed && mem_offset == rep->hdr_sz)) {
```

Comment:
Can you explain the logic here? What does the left part of the condition mean? What does the right part of the condition mean? This question is not meant as a dissatisfaction with the proposed change. I am just trying to understand (how incremental parsing affects) this mysterious condition.